index.html

```diff
@@ -1,25 +1,44 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <link rel="stylesheet" href="style.css">
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Capstone</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+    <title>Capstone</title>
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
   <nav class="nav-box">
-    <div class="hamburger">
+    <div class="hamburger" onclick="toggleMenu()">
       <span></span>
       <span></span>
       <span></span>
     </div>
+    <div class="dropdown-menu">
+      <a href="#home">Home</a>
+      <a href="#services">Services</a>
+      <a href="#musicians">Resident-Musicians</a>
+       <a href="about.html">About</a>
+    </div>
+     <div class="logo">
+        <img src="Images/Logo.png" alt="logo">
+      </div>
+    <div class="dropdown-desktp">
+      <a href="#home">Home</a>
+      <a href="#services">Services</a>
+      <a href="#musicians">Resident-Musicians</a>
+       <a href="about.html">About</a>
+    </div>
+       
     
+      
 
   </nav>
-  <section>
+  <section id="home">
     <div class="whole-body1">
```

Comment:
Great design @God-Logic 🤝

- [ ] Kindly ensure your design replicates the design given. You may switch the `hx's for these two sections. Kindly use an `h3` for the first heading and an `h1` for the second heading.
- Your design
![your-fonts](https://user-images.githubusercontent.com/63917725/233805541-7015069c-2885-4494-b835-38baf1194ec3.PNG)
- Given design
![given-fonts](https://user-images.githubusercontent.com/63917725/233805550-0b19c40c-1887-4859-9fda-f3a496ab1d94.PNG)